elasticsearch/src/main/scala/com/snowplowanalytics/stream/loader/ElasticsearchStreamLoaderApp.scala

```diff
@@ -0,0 +1,125 @@
+/**
+  * Copyright (c) 2014-2017 Snowplow Analytics Ltd.
+  * All rights reserved.
+  *
+  * This program is licensed to you under the Apache License Version 2.0,
+  * and you may not use this file except in compliance with the Apache
+  * License Version 2.0.
+  * You may obtain a copy of the Apache License Version 2.0 at
+  * http://www.apache.org/licenses/LICENSE-2.0.
+  *
+  * Unless required by applicable law or agreed to in writing,
+  * software distributed under the Apache License Version 2.0 is distributed
+  * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+  * either express or implied.
+  *
+  * See the Apache License Version 2.0 for the specific language
+  * governing permissions and limitations there under.
+  */
+
+package com.snowplowanalytics.stream.loader
+
+// Scalaz
+import scalaz._
+import Scalaz._
+
+// This project
+import clients.BulkSender
+import clients.ElasticsearchBulkSender
+import transformers.{BadEventTransformer, JsonTransformer, PlainJsonTransformer}
+import executors.{KinesisSourceExecutor, NsqSourceExecutor}
+import model._
+
+/** Main entry point for the Elasticsearch HTTP sink */
+object ElasticsearchStreamLoaderApp extends StreamLoaderApp {
+  val bulkSender: BulkSender[EmitterJsonInput] = config.elasticsearch match {
+    case None =>
+      throw new RuntimeException("No configuration for Elasticsearch found.")
+    case _ =>
+      new ElasticsearchBulkSender(
+        config.elasticsearch.get.client.endpoint,
+        config.elasticsearch.get.client.port,
+        config.elasticsearch.get.client.ssl,
+        config.elasticsearch.get.aws.region,
+        config.elasticsearch.get.aws.signing,
+        config.elasticsearch.get.client.username,
+        config.elasticsearch.get.client.password,
+        config.elasticsearch.get.client.maxTimeout,
+        config.elasticsearch.get.client.maxRetries,
+        config.elasticsearch.get.cluster.documentType,
```

Comment:
`get` shouldn't be needed, you case should be `case Some(esConfig)` => esConfig.client.endpoint`